Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Function.cs

```diff
@@ -20,19 +22,23 @@ public Functions()
     /// <summary>
     /// A Lambda function to respond to HTTP Get methods from API Gateway
     /// </summary>
-    /// <param name="request"></param>
-    /// <returns>The API Gateway response.</returns>
-    public APIGatewayProxyResponse Get(APIGatewayProxyRequest request, ILambdaContext context)
+    /// <remarks>
```

Comment:
We should have a `LambdaStartup` class with comments talking about how it can be used to register the DI. The `ConfigureServices` method should be empty.

Reply:
Lifted the empty `Startup.cs` from the annotations template, added to both EmptyServerless templates.